acm/src/main/java/org/literacybridge/acm/store/MetadataStore.java

```diff
@@ -43,4 +60,14 @@ public final void commit(Committable... objects) throws IOException {
         t.addAll(objects);
         t.commit();
     }
+
+    public interface DataChangeListener {
```

Comment:
I wonder if one of the java.lang.* events would be appropriate here? Maybe not; Sun tended to make things too concrete from the start.